.tekton/main-build.yaml

```diff
@@ -42,7 +42,7 @@ spec:
   - name: prefetch-input
     value: |
       [
-        { "type": "npm", "path": ".konflux/bootstrap-yarn" },
+        { "type": "npm", "path": "ui/apps/platform" },
         { "type": "gomod", "path": "." }
       ]
   - name: build-source-image
```

Comment:
I cannot comment on the particular line. The one of interest is 62, `spec.timeouts.pipeline`.
Please try bump it to 2 hours or so. From the run <https://console.redhat.com/preview/application-pipeline/workspaces/rh-acs/applications/acs/pipelineruns/main-build-qw5tj>, nothing seemed to fail. Just the overall execution took more than 1.5 hours.

I sort-of expected longer run times because I noticed that `prefetch-dependencies` for Go takes more than go build without prefetch (i.e. dependencies are downloaded from the internet). Not surprised that prefetching for NPM also adds time.
Cachi2, despite the name, did not (perhaps still does not) cache dependencies for faster access. Right now, it solves the supply chain trust problem (although I don't know if that's doable without persistence).

Besides `prefetch-dependencies`, I see longer time went into the `build-source-image` which is explainable if that includes source of more dependencies (at least the task pulls that via `CACHI2_ARTIFACT` parameter).

Sometimes our pipelines are just slow because we run into transient quota issues but either way I see no harm to bump the timeout.

Reply:
Thanks for the review Misha!

I'll give this a shot - I've noticed that the konflux main-build jobs have been failing, but honestly I haven't put a ton of effort into investigating why. Now that you pointed it out, I do see that the jobs are all failing at 1.5 hours exactly so I bet increasing the execution time limit will fix it.